changelog.json

```diff
@@ -670,8 +670,7 @@
       "[Fixed] 'Create Without Pushing' button throwing an exception while opening a pull request - #2368",
       "[Fixed] Windows: Don't removing the running app out from under itself when there are updates pending - #2373",
       "[Fixed] Windows: Respect `core.autocrlf` and `core.safeclrf` when modifying the .gitignore - #1535",
-      "[Fixed] Windows: Fix opening the app from the command line - #2396",
-      "[Fixed] 'Create Without Pushing' button throwing an exception while opening a pull request - #2368"
+      "[Fixed] Windows: Fix opening the app from the command line - #2396"
```

Comment:
Just curious why this change is here. Didn't this get fixed when we pushed the release a few days ago?

Reply:
This was caught because we can validate that the array of strings in a release entry are unique:

![](https://user-images.githubusercontent.com/359239/37941020-95fabe06-31b7-11e8-99f3-472c6b7d827d.png)

I figured this would be helpful to catch duplicate entries from appearing in the changelog, but it's not something I've got strong feelings about.

Reply:
oh, I see. I'm all good for keep this in 👍